RELEASE.md

```diff
@@ -52,6 +52,29 @@ If a bug fix got accidentally merged into master after non-bug-fix changes in ma
 
 Maintaining the release branches for older minor releases happens on a best effort basis.
 
+### Updating dependencies
+
+A few days before a major or minor release, consider updating the dependencies:
+
+```
+export GO111MODULE=on
+go get -u ./...
+go mod tidy
+go mod vendor
+git add go.mod go.sum vendor
+```
+
+Then create a pull request against the master branch.
+
+Note that after an update of the dependencies, you should look out for any
+weirdness that might have happened. Such weirdnesses include but are not limited
+to: flaky tests, differences in resource usage, panic.
+
+In case of doubt or issues that can't be solved in a reasonable amount of time,
+you can skip the dependencies update or only update select dependencies. In such
```

Comment:
Super sorry, but here is another "dependencies update". I'd say "dependency update" here, but a native speaker might have an even better idea.

Reply:
Yeah, dependency update is what I'd say here. It's a car sale, not a cars sale.